map/bookmark_helpers.cpp

```diff
@@ -479,7 +479,10 @@ bool SaveKmlFile(kml::FileData & kmlData, std::string const & file, KmlFileType
   try
   {
     FileWriter writer(file);
-    success = SaveKmlData(kmlData, writer, fileType);
+    if (fileType != KmlFileType::Gpx)
```

Comment:
Let's make a switch here and in other related places. There is also geojson coming.

Reply:
Replaced with switch